### ISSUE TITLE
cargo build - "Could not compile `neovim-rs`." under OS X 10.11.3

### ISSUE THREAD

**Issue by xale** [oakes/SolidOak#47]
Build output here: http://paste.openstack.org/show/485548/

I'm following the (homebrew) instructions in the README, and everything up to the 'cargo build' succeeds.

I've Googled the error message and found a couple of similar reports, but none of the workarounds/solutions seem to apply.

Curiously, cloning and building the neovim repository directly works.

Any idea what I'm missing here?

**Comment by oakes** [comment]
Hmm, this is the same issue reported [here](https://github.com/oakes/SolidOak/issues/46#issuecomment-170426285), but that user said my changes fixed it. It may be a transient issue that goes away after some particular thing happens. I'll have to keep looking into this.

**Comment by xale** [comment]
One thing from the README/build instructions just jumped out at me... there's a note about running `brew install vte3.rb` "directly in the project", (which I assume means "in the base directory of the repo") but Homebrew seems to have installed it in a common location, rather than local to the directory, since I just get an "already installed" message if I run it again after deleting and re-cloning the repo. Could this be related somehow?

**Comment by xale** [comment]
...aaaaaand of course, having cloned the repo a second time and run a fresh build, it compiles without errors.

Another theory about what may have gone wrong: when I first cloned the repo, I assumed I had the Apple "Command Line Tools" installed, when I apparently did it. Maybe I generated some weird build artifacts by trying to start a build without them?

Starting over after installing them seems to have been the solution.
